README.md

```diff
@@ -88,8 +88,7 @@ automatically optimize it for production-ready environments.
 Towhee requires Python 3.6+. Towhee can be installed via `pip`:
 
 ```bash
-% pip install -U pip  # if you run into installation issues, try updating pip
-% pip install towhee towhee.models
+pip install towhee towhee.models
```

Comment:
We need to provide a solution if the user runs into any trouble. Can we move the pip upgrade into a separate code block?
```
If you run into any pip-related install problems, please try to upgrade pip with `pip install -U pip`
```